### ISSUE TITLE
verbose message when connecting to all

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#28]
rvc -c now returns 'code 0' but we'd like a little bit more verbosity :)

```sh
$ ./rvc -c all -j | jq .
{
  "code": 0
}
```

Can you display something like this: (showing only connections that are triggered to "connecting")
``` json
{
  "status": 0,
  "connections": [{
    "name": "myvpn-connection-1",
    "status": "connecting",
    "status-since": "2017-08-02T03:11:52Z",
    "timestamp": "2017-08-02T05:03:16Z"
  },
  ...
  ]
}
```

And in case of no VPNs (or all VPNs are already connected) something like:

``` json
{
  "status": 0,
  "connections": []
}
```

**Comment by ronaldtse** [comment]
Updated expected response 😉

**Comment by jjr840430** [comment]
Fixed on #52
